profiles/base.yaml

```diff
@@ -61,6 +61,7 @@ dance_actions:
 - circle
 - bob
 - weave
+use_tactics_combos: false
```

Comment:
I think this setting should be next to `use_barb_combos`. I don't know which one should move to be near the other. What are your thoughts?